internal/entities/command_search.go

```diff
@@ -72,9 +72,17 @@ The search command performs a search for New Relic entities.
 			if len(entityFields) > 0 {
 				mapped := mapEntities(entities, entityFields, utils.StructToMap)
 
-				json, err = prettyjson.Marshal(mapped)
+				if len(mapped) == 1 {
```

Comment:
This conditionally changes the return type.  Could that cause problems for users?

Reply:
The argument here was that we should not return an array for a single item result.  We've discussed both sides of it, I don't have a strong leaning either way, but put it in anyway.